.prettierignore

```diff
@@ -6,81 +7,81 @@ packages/design-system
 !packages/nodes-base/test
 !packages/nodes-base/nodes
 
-packages/nodes-base/nodes/ActionNetwork
-packages/nodes-base/nodes/ActiveCampaign
-packages/nodes-base/nodes/AcuityScheduling
-packages/nodes-base/nodes/Affinity
-packages/nodes-base/nodes/AgileCrm
-packages/nodes-base/nodes/Airtable
-packages/nodes-base/nodes/Amqp
-packages/nodes-base/nodes/ApiTemplateIo
-packages/nodes-base/nodes/Asana
-packages/nodes-base/nodes/Automizy
-packages/nodes-base/nodes/Autopilot
-packages/nodes-base/nodes/Aws
-packages/nodes-base/nodes/BambooHr
-packages/nodes-base/nodes/Bannerbear
-packages/nodes-base/nodes/Baserow
-packages/nodes-base/nodes/Beeminder
-packages/nodes-base/nodes/Bitbucket
-packages/nodes-base/nodes/Bitly
-packages/nodes-base/nodes/Bitwarden
-packages/nodes-base/nodes/Box
-packages/nodes-base/nodes/Brandfetch
-packages/nodes-base/nodes/Bubble
-packages/nodes-base/nodes/Cal
-packages/nodes-base/nodes/Calendly
-packages/nodes-base/nodes/Chargebee
-packages/nodes-base/nodes/CircleCi
-packages/nodes-base/nodes/Cisco
-packages/nodes-base/nodes/Clearbit
-packages/nodes-base/nodes/ClickUp
-packages/nodes-base/nodes/Clockify
-packages/nodes-base/nodes/Cockpit
-packages/nodes-base/nodes/Coda
-packages/nodes-base/nodes/CoinGecko
-packages/nodes-base/nodes/Compression
-packages/nodes-base/nodes/Contentful
-packages/nodes-base/nodes/ConvertKit
-packages/nodes-base/nodes/Copper
-packages/nodes-base/nodes/Cortex
-packages/nodes-base/nodes/CrateDb
-packages/nodes-base/nodes/Cron
-packages/nodes-base/nodes/Crypto
-packages/nodes-base/nodes/CustomerIo
-packages/nodes-base/nodes/DateTime
-packages/nodes-base/nodes/DeepL
-packages/nodes-base/nodes/Demio
-packages/nodes-base/nodes/Dhl
-packages/nodes-base/nodes/Discord
-packages/nodes-base/nodes/Discourse
-packages/nodes-base/nodes/Disqus
-packages/nodes-base/nodes/Drift
-packages/nodes-base/nodes/Dropbox
-packages/nodes-base/nodes/Dropcontact
-packages/nodes-base/nodes/EditImage
-packages/nodes-base/nodes/Egoi
-packages/nodes-base/nodes/Elastic
-packages/nodes-base/nodes/EmailReadImap
-packages/nodes-base/nodes/EmailSend
-packages/nodes-base/nodes/Emelia
-packages/nodes-base/nodes/ERPNext
-packages/nodes-base/nodes/ErrorTrigger
-packages/nodes-base/nodes/Eventbrite
-packages/nodes-base/nodes/ExecuteCommand
-packages/nodes-base/nodes/ExecuteWorkflow
-packages/nodes-base/nodes/Facebook
-packages/nodes-base/nodes/Figma
-packages/nodes-base/nodes/FileMaker
-packages/nodes-base/nodes/Flow
-packages/nodes-base/nodes/FormIo
-packages/nodes-base/nodes/Formstack
-packages/nodes-base/nodes/Freshdesk
-packages/nodes-base/nodes/Freshservice
-packages/nodes-base/nodes/FreshworksCrm
-packages/nodes-base/nodes/Ftp
-packages/nodes-base/nodes/Function
-packages/nodes-base/nodes/FunctionItem
+# packages/nodes-base/nodes/ActionNetwork
```

Comment:
On completion, let's remove instead of commenting out.

Reply:
done